package.json

```diff
@@ -36,53 +23,68 @@
   ],
   "author": "dthree",
   "license": "MIT",
+  "scripts": {
+    "compile": "tsc -p ./",
+    "watch": "tsc -watch -p ./",
+    "lint:base": "tslint -p tsconfig.json",
```

Comment:
FYI, `tslint` is going to be deprecated soon ("sometime in 2019").
/ref https://github.com/palantir/tslint#tslint

Reply:
ack

Reply:
?

Reply:
acknowledged :)

Reply:
Ooh, that make sense :).